website/docs/pact_nirvana/step_5.md

```diff
@@ -37,34 +37,34 @@ By using branches alongside application versions in the broker allows you to sep
 
 To achieve this, when a pact is published, the associated pacticipant version should be published with a branch identifier that will be used by the provider to differentiate between the stable, safe pacts \(eg. branch "master"\) and the potentially breaking pacts \(eg. branch "feat-new-foobar"\).
 
-To keep a green build, in your provider’s CI, we will utilise consumer version selectors to verify the main branch, rather than verifying the latest overall pact.
+To maintain a green build in your provider’s CI, we will use consumer version selectors to verify the main branch rather than the latest overall pact.
 
-If you use feature branches for your consumer development, it is recommended to publish the pact with pacticipant version and the name of the branch. If you use feature toggles, the branch could be the name of the feature toggle. Your Pact client library will allow you to configure the name of the branch to be applied when you publish your pacts.
+If you use feature branches for your consumer development, it is recommended to publish the pact with the participant version and the name of the branch. If you use feature toggles, the branch could be the name of the feature toggle. Your Pact client library will allow you to configure the name of the branch to be applied when you publish your pacts.
```

Comment:
I think this should possibly remain as `pacticipant`

and possibly link the terminology section for the word here, if you feel the reader may be confused here

https://docs.pact.io/getting_started/terminology#pacticipant

wdyt?

Reply:
Alright, now I understand where that is coming from.
When I read that the first time, I thought it was a typo, and someone was typing "pact" just a few times too often before that.
I'm okay with both options but prefer the word **participant** as it has the same semantics but is an actual word

Reply:
I just changed it back to pacticipant and linked the definition.
Thansk for the review!

Reply:
>When I read that the first time, I thought it was a typo, and someone was typing "pact" just a few times too often before that.

It happens to us all! Thanks, the link to the definition will certainly help.

Maybe setting some of the terminology at the start of guide might be helpful.

Appreciate the edits!